toolkit/build.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+TOBUILD=
+BID=lkg
+RC=n
+WCREFRESH=n
+SDIR=../SPECS
+USEPKGBLDCACHE=y
+
+helpFuncion () {
+	echo ""
+	echo "-h  : Print detailed usage"
+        echo "-s  : Spec Dir [Default SPECS] [Possible values: ../SPECS-EXTENDED, ../SPECS]"
+        echo "-b  : DAILY_BUILD_ID of the format of 3-0-YYYYMMDD [Default 'lkg']"
+        echo "-c  : Run with check distabled/enabled [Default RUN_CHECK=n]"
+        echo "-p  : Package to build/rebuild [Default all]"
+        echo "-r  : Refresh worker chroot [Default false]"
+        echo "-f  : Force a Rebuild"
+        echo "-n  : Cleanup input-srpms expand-srpms"
+        echo "-d  : Show the command to be executed."
+	echo ""
+	echo ""
+	echo "Examples:"
+	echo "[inside the toolkit dir]"
+	echo "Build vim & golang locally:"
+	echo "./build.sh -p \"vim golang\""
+	echo ""
+	echo "Build cpptest from extended:"
+	echo "./build.sh -s=../SPECS-EXTENDED -p cpptest"
+	echo ""
+}
+
+
+while getopts "hs:b:cp:rfnd" opt
+do
+	case "$opt" in
+		h ) helpVar=y;;
+		s ) specsVar="$OPTARG";;
+		b ) buildidVar="$OPTARG";;
+		c ) checkVar=y;;
+		p ) packageVar="$OPTARG";;
+		r ) refreshVar=y;;
+		f ) forceVar=y;;
+		n ) nukeVar=y;;
+		d ) dryrunVar=y;;
+	esac
+done
+
+if [ -n "$helpVar" ]
+then
+	helpFuncion
+	exit 0
+fi
+
+if [ -n "$specsVar" ]
+then
+	SDIR=$specsVar
+fi
+
+if [ -n "$buildidVar" ]
+then
+	BID="$buildidVar"
+fi
+
+if [ -n "$checkVar" ]
+then
+	RC=y
+fi
+
+if [ -z "$packageVar" ]
+then
+	echo "Package var needs to be set:";
```

Comment:
I would reword this to 
`Package to build needs to be set`
to avoid looking up what Package _var_ refers to

Reply:
Good catch.